docs/build/iota-sdk/1.0/docs/getting-started/wasm.mdx

```diff
@@ -147,51 +156,69 @@ A bundler such as [webpack](https://webpack.js.org/) or [rollup](https://rollupj
 
 After you [installed the library](#install-the-iota-sdk), you can create a `Client` instance and interface with it.
 
+The following example creates a [`Client`](https://wiki.iota.org/shimmer/iota-sdk/references/nodejs/classes/Client/)
+instance connected to
+the [Shimmer Testnet](https://api.testnet.shimmer.network), and retrieves the node's information by
+calling [`Client.getInfo()`](https://wiki.iota.org/shimmer/iota-sdk/references/nodejs/classes/Client/#getinfo),
+and then print the node's information.
+
+<Tabs groupId="usage" queryString>
+<TabItem value="Node.js" label="Node.js">
+<div className={'hide-code-block-extras'}>
+
 ```typescript reference
-https://github.com/iotaledger/iota-sdk/blob/develop/bindings/nodejs/examples/client/getting-started.ts#L4-L23
+https://github.com/iotaledger/iota-sdk/blob/develop/bindings/wasm/README.md#L127-L145
 ```
 
+</div>
+</TabItem>
+<TabItem value="Web" label="Web">
+<div className={'hide-code-block-extras'}>
+
+```typescript reference
+https://github.com/iotaledger/iota-sdk/blob/develop/bindings/wasm/README.md#L151-L168
+```
+
+</div>
+</TabItem>
+</Tabs>
+
 ### Wallet
 
 After you [installed the library](#installing-the-iota-sdk), you can create a `Wallet` instance and interact with it.
 
+The following example will create a
+new [`Wallet`](https://wiki.iota.org/iota-sdk/references/nodejs/classes/Wallet/) [`Account`](https://wiki.iota.org/iota-sdk/references/nodejs/classes/Account/)
+that connects to the [Shimmer Testnet](https://api.testnet.shimmer.network) using the
+[`MnemonicSecretManager`](https://wiki.iota.org/iota-sdk/references/nodejs/interfaces/MnemonicSecretManager/)
+by calling
+the [`Wallet.createAccount(data)`](https://wiki.iota.org/iota-sdk/references/nodejs/classes/Wallet/#createaccount)
+function.
+
+
+<Tabs groupId="usage" queryString>
+<TabItem value="Node.js" label="Node.js">
+<div className={'hide-code-block-extras'}>
```

Comment:
Do we want to compare with node.js code here? IF so, was/web should be first and default tho @lucas-tortora

Reply:
I don't think I understood what you mean. Why should Web be first?